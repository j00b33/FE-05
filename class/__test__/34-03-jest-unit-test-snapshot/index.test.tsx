import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import JestUnitTestSnapshotPage from "../../pages/34-03-jest-unit-test-snapshot";

it("내가 원하는대로 그려지는지 테스트하기", () => {
  const result = render(<JestUnitTestSnapshotPage />);
  expect(result.container).toMatchSnapshot();
  //없으면 스냅샷 찍고 있으면 기존 스냅샷과 비교
});
