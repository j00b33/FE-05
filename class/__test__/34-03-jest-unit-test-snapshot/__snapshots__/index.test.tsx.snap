// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`내가 원하는대로 그려지는지 테스트하기 1`] = `
<div>
  <div>
    <div>
      철수는 13살 입니다
    </div>
    철수의 취미 입력하기: 
    <input
      type="text"
    />
    <button>
      철수랑 놀러가기
    </button>
  </div>
</div>
`;
